games-server/crossfire-server/crossfire-server-1.75.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="${P/-server/}"
+DESCRIPTION="Server for the crossfire clients"
+HOMEPAGE="http://crossfire.real-time.com/"
+SRC_URI="https://downloads.sourceforge.net/crossfire/crossfire-server/${PV}/crossfire-${PV}.tar.gz
+	https://downloads.sourceforge.net/crossfire/crossfire-maps/${PV}/crossfire-maps-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="X"
+RESTRICT="test"
+
+RDEPEND="
+	net-misc/curl
+	sys-libs/zlib
+	virtual/libcrypt:=
+	X? (
+		x11-libs/libXaw
+		media-libs/libpng:0=
+	)
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	default
+
+	# bug #236205
+	rm -f "${WORKDIR}"/maps/Info/combine.pl || die
+#	ln -s "${WORKDIR}"/arch "${S}"/lib || die
+
+	eapply "${FILESDIR}"/${P}-format.patch
+}
+
+src_configure() {
+	econf --disable-static
+}
```

Comment:
I think `src_configure` could be removed entirely. `EAPI8` already pass `--disable-static` per default (if it's available in the configure script)